src/sagemaker/jumpstart/types.py

```diff
@@ -1253,18 +1115,1090 @@ def from_json(self, json_obj: Dict[str, Any]) -> None:
+        self.hosting_prepacked_artifact_uri: Optional[str] = (
+            s3_path_join("s3://", content_bucket, model_specs.hosting_prepacked_artifact_key)
+            if model_specs.hosting_prepacked_artifact_key is not None
+            else None
+        )
+        self.hosting_prepacked_artifact_version: Optional[str] = None  # TODO: Not in specs?
+        self.hosting_use_script_uri: Optional[bool] = model_specs.hosting_use_script_uri
+        self.hosting_eula_uri: Optional[str] = (
+            s3_path_join("s3://", content_bucket, model_specs.hosting_eula_key)
+            if model_specs.hosting_eula_key is not None
+            else None
+        )
+        self.hosting_model_package_arn: Optional[str] = model_specs.hosting_model_package_arns
+        self.default_inference_instance_type: Optional[
+            str
+        ] = model_specs.default_inference_instance_type
+        self.supported_inference_instance_types: Optional[
+            str
+        ] = model_specs.supported_inference_instance_types
+        self.sage_maker_sdk_predictor_specifications: Optional[str] = model_specs.predictor_specs
+        self.inference_volume_size: Optional[int] = model_specs.inference_volume_size
+        self.inference_enable_network_isolation: Optional[
+            str
+        ] = model_specs.inference_enable_network_isolation
+        self.fine_tuning_supported: Optional[bool] = studio_specs.get("fineTuningSupported")
+        self.validation_supported: Optional[bool] = studio_specs.get("validationSupported")
+        default_training_dataset_key = studio_specs.get("defaultDataKey")
+        self.default_training_dataset_uri: Optional[str] = (
+            s3_path_join("s3://", content_bucket, model_specs.hosting_prepacked_artifact_key)
+            if default_training_dataset_key
+            else None
+        )
+        self.resource_name_base: Optional[str] = model_specs.resource_name_base
+        self.gated_bucket: bool = model_specs.gated_bucket
+        self.default_payloads: Optional[
+            Dict[str, JumpStartSerializablePayload]
+        ] = model_specs.default_payloads
+        self.hosting_resource_requirements: Optional[
+            Dict[str, int]
+        ] = model_specs.hosting_resource_requirements
+        self.hosting_instance_type_variants: Optional[
+            str
+        ] = model_specs.hosting_instance_type_variants
+        self.min_studio_sdk_version: Optional[str] = studio_specs.get("minServerVersion")
+        notebook_location_keys = studio_specs.get("notebookLocations")
+        if notebook_location_keys:
+            self.notebook_location_uris: Optional[Dict[str, str]] = (
+                s3_path_join("s3://", content_bucket, notebook_location_keys.get("demoNotebook"))
+                if notebook_location_keys.get("demoNotebook")
+                else None
+            )
+        self.model_provider_icon_uri: Optional[str] = None  # TODO: Missing in specs?
```

Comment:
This is a UI field, we need to upload these somewhere, let's chat offline w/ @bencrabtree